<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTCWindow</class>
 <widget class="QMainWindow" name="CTCWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>683</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CTC Module</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_chooseSchedule">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Schedule File...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_upcomingSchedule">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Upcoming Schedule</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_mode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_ctcDispatchMode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CTC Dispatch Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_autoDispatchMode">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_manualDispatchMode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_schedule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_manualDispatchGroup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Manual Dispatch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_manualDispatchtrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_manualDispatchTo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Destination</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_manualDispatchToTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dispatchTrain">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dispatchTo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dispatchTime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dispatchTrain">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>181</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dispatch Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_selectBlock">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Block</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_selectBlock">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_blockProperties">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Block Properties</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_blockPropertiesId">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Block #:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_blockPropertiesId">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_blockPropertiesMaintenence">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Maintenence:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_blockPropertiesMaintenence">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_blockPropertiestrainPresent">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Train Present:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_blockPropertiesTrainPresent">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_blockPropertiesThroughput">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Throughput:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_blockPropertiesThroughput">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openCloseForMaintenecnce">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>221</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Open/Close for Maintenence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_selectBlockGo">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>401</y>
      <width>491</width>
      <height>211</height>
     </rect>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_selectSwitch">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Switch</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>211</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_switchProperties">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Switch Properties</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_switchPropertiesId">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Block #:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_switchId">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_switchDirection">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>67</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_switchDirection">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openCloseSwitch">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>221</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Open/Close Switch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trackMap">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Track Map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChoose_Schedule_File">
   <property name="text">
    <string>Choose Schedule File...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
